data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
```

Comment:
These shield and hull values seem pretty round.

Reply:
Um. I like round numbers? Not sure if this actually needs fixing.

Reply:
>I like round numbers?

Factions outside of Republic space don't tend to have round numbers. :)

Reply:
OK, I'll break them up a bit.